### ISSUE TITLE
No more hand cursor on dropdown carret

### ISSUE THREAD

**Issue by phiphou** [openstreetmap/iD#5769]
Hi,

From 2.13.0, there is no more hand cursor on dropdown carret.

**Comment by bhousel** [comment]
Thanks for reporting @phiphou - the carat now is more for decoration than anything.  Users can click anywhere on the field to expand the dropdown.  I'll see whether we can change the cursor.

**Comment by phiphou** [comment]
Yes, I want the current behavior (click anywhere), cursor is not important, but if you can change it, it will be more beautiful.
